docs/publishers/config.rst

```diff
@@ -167,6 +187,22 @@ loads.
 
      This setting can only be set using :js:func:`window.hypothesisConfig`.
 
+   .. option:: onProfileRequest
+
+     ``function``. A JavaScript function that will be called when the user clicks
+     the user profile (user name) link in the sidebar. No arguments are passed
```

Comment:
Yeah, URL templates might be good. The rendered could even be part of the annotation JSON objects returned by the annotation API, so an API client wouldn't even need the links API.

It still does have the downside that there won't be a way to link to a normal Hypothesis profile page for an `@elife.org` user, so they'll miss out on any benefits and features that we add to our profile pages. (Unless we render _both_ profile links on the annotation card, our render just our own but have our profile page contain a link to the publisher's profile page...)

I think it needs discussion